assets/chars.json

```diff
@@ -417,54 +417,106 @@
+    "height": "170 cm (5'7\")",
+    "weight": "59 kg (130 lbs)",
+    "blood_type": "B",
     "emoteId": "909168184551157830",
     "cuteEmoteId": "909168184551157830"
   },
   {
     "name": "Lloyd Sirus",
-    "age": "??",
+    "age": 30,
     "description": "Lloyd is a high-energy architect who's often teased for his short stature. He can be eccentric at times when it comes to his peculiar hobbies.",
-    "birthday": "??",
-    "animal": "??",
+    "birthday": "March 21",
+    "animal": "Hamster",
     "color": "#7CA2D1",
-    "height": "??",
-    "weight": "??",
-    "blood_type": "??",
+    "height": "158 cm (5'2\")",
+    "weight": "57 kg (125 lbs)",
+    "blood_type": "B",
     "emoteId": "909166826234212412",
     "cuteEmoteId": "909166826234212412"
   },
   {
     "name": "Darius Najjar",
-    "age": "??",
+    "age": 29,
     "description": "Darius is a stoic construction foreman who's a man of few words. While his cold and mystrious demeanor can be intimidating, he has a kind heart and a very dry sense of humor.",
-    "birthday": "??",
-    "animal": "??",
-    "color": "#6B0408",
-    "height": "??",
-    "weight": "??",
-    "blood_type": "??",
+    "birthday": "November 1",
+    "animal": "Giraffe",
+    "color": "#6b0408",
+    "height": "193 cm (6'4\")",
+    "weight": "108 kg (238 lbs)",
+    "blood_type": "A",
     "emoteId": "909168172073103361",
     "cuteEmoteId": "909168172073103361"
+  },
+  {    
+    "name": "Justin",
+    "age": 21,
+    "description": "As the lead singer of a pop-punk band, Justin loves to perform for others! A chaotic tease with confidence, and he wants his songs to shine bright, for those who need a light to guide their way.",
+    "birthday": "July 31",
+    "animal": "Beaver",
+    "color": "#b31ebc",
+    "height": "170 cm (5'7\")",
+    "weight": "58 kg (127 lbs)",
+    "blood_type": "O-",
+    "emoteId": "946971084119482368",
+    "cuteEmoteId": "946971084119482368"
+  },
+  {    
+    "name": "Reimond",
+    "age": 19,
+    "description": "You can see him everywhere, Reimond is able to completely satisfy you in the Grand O Hotel. From the spa to room service, or even... extra service, he is at your beck and call! Due to the stress from over-working himself, he might have developed some twisted personality...",
+    "birthday": "October 22",
+    "animal": "Nightingale",
+    "color": "#e1b433",
+    "height": "172 cm (5'7\")",
+    "weight": "60 kg (132 lbs)",
+    "blood_type": "A",
+    "emoteId": "957014101480079471",
+    "cuteEmoteId": "957014101480079471"
+  },
+  {    
+    "name": "Vera Nomoru",
+    "age": 38,
+    "description": "",
+    "birthday": "November 14",
+    "animal": "None",
+    "color": "#d52aa5",
+    "height": "170 cm (5'7\")",
+    "weight": "68 kg (149 lbs)",
+    "blood_type": "A",
+    "emoteId": "833333254634602496",
+    "cuteEmoteId": "833333254634602496"
+  },
+  {    
+    "name": "Andre Flynn",
+    "age": 36,
+    "description": "",
```

Comment:
Is it ok to leave these as blank strings? I've looked but there appears to be no real description for them anywhere. Otherwise would "???" be ok?

Reply:
The strings are safe to leave empty, but the types are expected to match what's given. A emote id should be a valid id for example

Reply:
Descriptions are either taken from an announcement post for the characters or are written by us based on what we learn about them in-game.